scripts/jenkins/groovy/insideDocker.groovy

```diff
@@ -8,8 +8,10 @@ def call(customEnv, image, registry, buildConfig, timeoutValue, timeoutUnit, cus
 
   retryWithDelay(3 /* retries */, 120 /* delay in sec */) {
     withCredentials([usernamePassword(credentialsId: registry, usernameVariable: 'REGISTRY_USERNAME', passwordVariable: 'REGISTRY_PASSWORD')]) {
-      sh "docker login -u $REGISTRY_USERNAME -p $REGISTRY_PASSWORD ${registry}"
-      sh "docker pull ${image}"
+      sh """
+docker login -u $REGISTRY_USERNAME -p $REGISTRY_PASSWORD ${registry}
```

Comment:
@michal-raska I think it's better to avoid indents inside values (filled from text snippets), it can be occasionally dangerous. But OK here - it is unlikely to affect the execution

Reply:
thx! I think it's safe here and it improves readability I think.